alpine/Dockerfile.diffblue-builder

```diff
@@ -6,7 +6,8 @@ RUN apk add --no-cache \
       openjdk8 \
       jq \
       maven \
-      apache-ant
+      apache-ant \
+      python
```

Comment:
Any reason not to sort these alphabetically?

Reply:
Done!